README.md

```diff
@@ -25,3 +25,12 @@ Preupgrade assistant performs assessment of the system from the "upgradeability"
 ## How to execute preupgrade-assistant
 
 Just run ./preupg. But with root priviledges because of preupg binary needs to have an access to all files.
+
+## Contribute
```

Comment:
Would it be possible to move this section to CONTRIBUTE.md file so we will not modify README.md later on.